README.md

```diff
@@ -104,6 +104,7 @@ Currently **officially** using Airflow:
 * [Lucid](http://luc.id) [[@jbrownlucid](https://github.com/jbrownlucid) & [@kkourtchikov](https://github.com/kkourtchikov)]
 * [Lyft](https://www.lyft.com/)[[@SaurabhBajaj](https://github.com/SaurabhBajaj)]
 * [Nerdwallet](https://www.nerdwallet.com)
+* [Qubole](https://qubole.com)[[@msumit](https://github.com/msumit)]
```

Comment:
Nit: a space between the company and person (you) would be good:

    [Qubole](https://qubole.com) [[@msumit](https://github.com/msumit)]